src/Xamarin.Android.NamingCustomAttributes/Android.App/ActivityAttribute.cs

```diff
@@ -50,32 +58,41 @@ public ActivityAttribute ()
 #if ANDROID_26
 		public float                  MaxAspectRatio          {get; set;}
 #endif
+		public int                    MaxRecents              {get; set;}
 		public bool                   MultiProcess            {get; set;}
 		public bool                   NoHistory               {get; set;}
 #if ANDROID_16
 		public Type                   ParentActivity          {get; set;}
 #endif
+		public ActivityPersistableMode      PersistableMode   {get; set;}
 		public string                 Permission              {get; set;}
 		public string                 Process                 {get; set;}
+		public bool                   RecreateOnConfigChanges {get; set;} 
+		public bool                   RelinquishTaskIdentity  {get; set;}
 #if ANDROID_24
 		public bool                   ResizeableActivity      {get;set;}
 #endif
+		public bool                   ResumeWhilePausing      {get; set;}
```

Comment:
This is not documented at: https://developer.android.com/guide/topics/manifest/activity-element